docs/contributing/Building, Debugging, and Testing on Unix.md

```diff
@@ -78,7 +70,7 @@ Note: this variables can also be used on Windows as well.
 
 ## Ubuntu 18.04
 The recommended OS for developing Roslyn is Ubuntu 18.04. This guide was written using Ubuntu 18.04 but should be 
-applicable to most Linux enviroments. Ubuntu 18.04 was chosen here due to it's support for enhanced VMs in Hyper-V. 
+applicable to most Linux environments. Ubuntu 18.04 was chosen here due to it's support for enhanced VMs in Hyper-V.
```

Comment:
For those keeping score at home, i mis-spelled environment three times in a single document with each having a different mis-spelling